src/blueprints/tv2/action-factories/test/tv2-transition-effect-action-factory.spec.ts

```diff
@@ -19,12 +20,14 @@ function createTestee(params?: {
   videoMixerTimelineObjectFactory?: Tv2VideoMixerTimelineObjectFactory,
   casparCgTimelineObjectFactory?: Tv2CasparCgTimelineObjectFactory,
   audioTimelineObjectFactory?: Tv2AudioTimelineObjectFactory,
-  assetPathHelper?: Tv2AssetPathHelper
+  assetPathHelper?: Tv2AssetPathHelper,
+  loggerService?: Tv2LoggerService
```

Comment:
I strongly prefer the logger is not suffixed with `Service`.

Reply:
`Service` has been removed from both the class name and attributes/argument names.